### PR TITLE
Sample's header table fields are not validated on submit

### PR DESCRIPTION
**Pull request by xispa**
## Description of the issue/feature this PR addresses

This Pull Request validates the values submitted for fields displayed in Sample's edit view and displays a message if errors are found.

## Current behavior before PR

The values for fields displayed in the Sample's edit view (header table) are not validated on submit

## Desired behavior after PR is merged

The values for fields displayed in the Sample's edit view (header table) are validated on submit

--
I confirm I have tested this PR thoroughly and coded it according to [PEP8][1]
and [Plone's Python styleguide][2] standards.

[1]: https://www.python.org/dev/peps/pep-0008
[2]: https://docs.plone.org/develop/styleguide/python.html

### CODE REVIEW

File: src/bika/lims/browser/analysisrequest/templates/analysisrequest_view.pt [hunk 1]
```diff
@@ -98,6 +98,20 @@
...
+            <h3>
+              <i class="labanalyses-icon" title="AST" i18n:attributes="title">
+                <svg tal:replace="structure senaite_theme/icon_data/analysisservice" />
+              </i>
+              <span i18n:translate="">Antibiotic Susceptibility</span>
+            </h3>
+            <span tal:replace="structure python:view.render_analyses_table(table='ast')"/>
+          </div>
+        </div>
+      </tal:ast_analyses>
```
Comment (ramonski) [inline src/bika/lims/browser/analysisrequest/templates/analysisrequest_view.pt:113]: This probably does not belong to the PR

**Review by ramonski — CHANGES_REQUESTED** [review]

File: src/bika/lims/browser/analysisrequest/templates/analysisrequest_view.pt (hunk 1 above)
Comment (xispa) [reply src/bika/lims/browser/analysisrequest/templates/analysisrequest_view.pt:113]: Yes apologies... this is the good one: https://github.com/senaite/senaite.core/pull/1687
